services/web/server/tests/unit/isolated/test_login_settings.py

```diff
@@ -0,0 +1,99 @@
+# pylint: disable=redefined-outer-name
+# pylint: disable=unused-argument
+# pylint: disable=unused-variable
+
+import os
+from typing import Any
+
+import pytest
+from models_library.errors import ErrorDict
+from pydantic import ValidationError
+from pytest import MonkeyPatch
+from pytest_simcore.helpers.utils_envs import setenvs_from_dict
+from settings_library.email import SMTPSettings
+from simcore_service_webserver.login.settings import LoginOptions, LoginSettings
+
+
+def test_login_with_invitation(monkeypatch: MonkeyPatch):
+    monkeypatch.setenv("LOGIN_REGISTRATION_INVITATION_REQUIRED", "1")
+
+    settings = LoginSettings.create_from_envs()
+    assert settings
+
+
+def test_settings_logging_with_2fa(monkeypatch: MonkeyPatch):
+
+    TWILO_CONFIG = {
+        "TWILIO_ACCOUNT_SID": "fake-account",
+        "TWILIO_AUTH_TOKEN": "fake-token",
+        "TWILIO_MESSAGING_SID": "x" * 34,
+    }
+    # NOTE: apparently some session-based fixtures is settings these envs
```

Comment:
hmm.. apparently?

Reply:
The problem here is that these variables are somehow already defined at the start of the test (even if no fixture was added) ... for that reason i have to explicitly *delenv* them!

I asy "apparently" because *my guess* is that ``monkeypatch_session`` and ``monkeypatch_module`` are somehow introducing these variables as a side-effect.  Those are very dangerous and marked for deprecation ... so I am slowly removing them